cairo-hints/scarb-hints-run/main.rs

```diff
@@ -69,11 +64,9 @@ struct Args {
     #[clap(long)]
     memory_file: Option<PathBuf>,
 
-    /// Arguments of the Cairo function.
-    #[clap(long = "args", default_value = "", value_parser=process_args)]
+    #[clap(long = "args", default_value = "", value_parser = process_args)]
     args: Option<FuncArgs>,
 
-    /// Arguments of the Cairo function.
     #[clap(long = "args_json", default_value = "")]
```

Comment:
Its a little late, but for CLI's the arguments are usually `kebab-case`. `--args-json` instead of `--args_json`

Reply:
It was in `snake_case`  original, before our fork. That's why I implemented like that. I will change it in a next PR. To change doc etc. as well